src/content/docs/implementation/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Headless storefront implementation
+description: Learn how to plan, develop, and launch a headless Adobe Commerce storefront with Edge Delivery Services.
+---
+
+import Callouts from '@components/Callouts.astro';
+import Diagram from '@components/Diagram.astro';
+
+If you're looking for guidance about launching a production-ready, headless Adobe Commerce storefront with [Edge Delivery Services](https://www.aem.live/), you're in the right place! This guide will help you successfully plan, implement, and launch your storefront.
+
+## Big picture
+
+Launching a headless Adobe Commerce storefront on Edge Delivery Services requires some basic planning and implementation steps before launch. Here's an overview of the process:
+
+<Diagram caption="Implementation planning and project delivery process.">
+  ![Implementation planning and project delivery process.](@images/implementation/implementation-overview-v1.svg)
+</Diagram>
+
+<Callouts>
+
+1. **Discovery**: Understand the requirements and goals of the project.
+1. **Setup**: Configure the project environment and tools.
+1. **Marketing and SEO**: Optimize the storefront for search engines and marketing campaigns.
+1. **Launch**: Deploy the storefront to a production environment.
+
+</Callouts>
+
+## Composability
+
+Headless Adobe Commerce storefronts on Edge Delivery Services are built using a composable architecture with domain-driven commerce components ([dropins](/dropins/all/introduction/)). This architecture allows you to build and deploy a storefront that is composed of multiple Adobe services, each with its own responsibility. Dropins are connected through APIs and can be developed, tested, and deployed independently for faster development cycles.
+
+Dropins rely on [Adobe Commerce](https://developer.adobe.com/commerce/webapi/graphql/) and [Catalog Service](https://developer.adobe.com/commerce/services/graphql/catalog-service/) APIs to provide data and functionality. These components are designed to be reusable and can be shared across multiple projects. They are integrated out-of-the-box with Edge Delivery Services through the Commerce boilerplate. Adobe Commerce storefronts on Edge Delivery Services are compatible with doc-based authoring, which allows business users to create and manage content without developer intervention.
+
+The following diagram illustrates the composable architecture of a headless Adobe Commerce storefront on Edge Delivery Services:
+
+<Diagram caption="Composable architecture of a headless Adobe Commerce storefront on Edge Delivery Services.">
+  ![Composable architecture of a headless Adobe Commerce storefront on Edge Delivery Services.](@images/implementation/headless-architecture-v2.svg)
```

Comment:
Where is CIF in this diagram coming from?

Reply:
I'm not sure. I adapted this diagram from @fnhipster's slide deck. Should I remove it?

Reply:
I haven't seen a single Edge Delivery storefront project that uses CIF. So please double check with Carlos if this should be in the public docs.

Reply:
@fnhipster, I'm going to remove CIF from the diagram. I can always add it back if you think it's necessary.